src/qml/editorwidgets/ExternalResource.qml

```diff
@@ -361,6 +363,15 @@ EditorWidgetBase {
 
   Connections {
     target: __viewStatus
+
+    onFinished: {
+      if (isImage) {
+        var imageSource = image.source;
+        image.source = '';
```

Comment:
Can we leave a comment like "This triggers a reload if the image content changed at the same path" (if this is actually correct)

Reply:
@m-kuhn , yep, it's correct, will add a comment (the reason why we do this is to trigger a refresh of the image to catch edited changes).

Reply:
Done.